moabb/datasets/martinezcagigal2023_checker_cvep.py

```diff
@@ -0,0 +1,322 @@
+    results are available in [1].
+
+    The encoding was displayed at a 120 Hz refresh rate. EEG signals were
+    recorded using a g.USBamp amplifier (g.Tec, Guger Technologies, Austria)
+    with 16 active electrodes and a sampling rate of 256 Hz. Electrodes were
+    placed at: Oz, F3, Fz, F4, I1, I2, C3, Cz, C4, CPz, P3, Pz, P4, PO7, POz,
+    PO8, grounded at AFz and referenced to the earlobe.
+
+    The experimental paradigm was executed using the MEDUSA© software [3],
+    with the publicly available application "c-VEP Speller":
+    https://www.medusabci.com/market/cvep_speller/
+
+    References
+    ----------
+    .. [1] Martínez Cagigal, V. (2025). Dataset: Influence of spatial frequency
+       in visual stimuli for cVEP-based BCIs: evaluation of performance and
+       user experience. https://doi.org/10.71569/7c67-v596
+
+    .. [2] Fernández-Rodríguez, Á., Martínez-Cagigal, V., Santamaría-Vázquez,
+       E., Ron-Angevin, R., & Hornero, R. (2023). Influence of spatial frequency
+       in visual stimuli for cVEP-based BCIs: evaluation of performance and user
+       experience. Frontiers in Human Neuroscience, 17, 1288438.
+       https://doi.org/10.3389/fnhum.2023.1288438
+
+    .. [3] Santamaría-Vázquez, E., Martínez-Cagigal, V., Marcos-Martínez, D.,
+       Rodríguez-González, V., Pérez-Velasco, S., Moreno-Calderón, S., &
+       Hornero, R. (2023). MEDUSA©: A novel Python-based software ecosystem to
+       accelerate brain–computer interface and cognitive neuroscience research.
+       Computer Methods and Programs in Biomedicine, 230, 107357.
+       https://doi.org/10.1016/j.cmpb.2023.107357
+
+    Notes
+    -----
+    Although the dataset was recorded in a single session, each condition is
+    stored as a separate session to match the MOABB structure. Within each
+    session, three MNE arrays are available (two for training, one for testing).
+
+    Due to limitations in the MNE format, both cycle onsets and event onsets
+    are stored as annotations, with labels included in their descriptions:
+        - "cycle_onset": marks the start of a stimulation cycle
+        - "0": marks the onset of a 0 (white) event
+        - "1": marks the onset of a 1 (black) event
+
+    The MNE format does not support encoding further paradigm-specific
+    information. For full access to the dataset's metadata and structure,
+    users are encouraged to load the original recordings using the MEDUSA Kernel.
+
+    Example:
+    >> pip install medusa-kernel
+    >> from medusa import components
+    >> from medusa.bci import cvep_spellers
+    >> rec = components.Recording.load(path)
+
+    The "rec" object will contain all available information.
+
+    .. versionadded:: 1.2.0
+    """
+
+    def __init__(self):
+        super().__init__(
+            subjects=list(range(1, len(SUBJECTS) + 1)),
+            sessions_per_subject=len(CONDITIONS),
+            events={},
```

Comment:
The `events` arg must be provided, otherwise the paradigms and benchmarks will not work with this dataset